internal/types/metadata_test.go

```diff
@@ -0,0 +1,39 @@
+package types
+
+import (
+	"testing"
+
+	"gotest.tools/assert"
+	is "gotest.tools/assert/cmp"
+)
+
+func TestMaintainer(t *testing.T) {
+	assert.Check(t, is.Equal(Maintainer{"foo", "foo@bar.com"}.String(), "foo <foo@bar.com>"))
+	assert.Check(t, is.Equal(Maintainer{"foo", ""}.String(), "foo <>"))
+	assert.Check(t, is.Equal(Maintainer{"foo", "bar"}.String(), "foo <bar>"))
+	assert.Check(t, is.Equal(Maintainer{"", ""}.String(), " <>"))
+}
```

Comment:
Now that you changed this test, you might want to do the same for the one after, and maybe for the ones in `internal/names_test.go`?

Reply:
:rabbit: